shell/platform/android/external_view_embedder/BUILD.gn

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/android/config.gni")
+import("//flutter/common/config.gni")
+import("//flutter/shell/config.gni")
+import("//flutter/shell/gpu/gpu.gni")
+import("//flutter/shell/version/version.gni")
+
+import("//flutter/testing/testing.gni")
+
+source_set("external_view_embedder") {
```

Comment:
It may be a good idea now to start thinking about unifying all the external view embedder implementations into a common utility that can be configured per the needs of specific embedders. Once fully realized, this is going to look very similar to the iOS and custom embedder implementations. And, at least the custom embedder implementation has a tests that may be useful for this effort.

Reply:
This is an excellent clean start to a clean room implementation however. So feel free to bring in the other stuff later.

Reply:
For sure. This change will help shape the next action items.  :)